modin/logging/config.py

```diff
@@ -36,6 +37,16 @@
 __LOGGER_CONFIGURED__: bool = False
 
 
+class LogLevel(IntEnum):  # noqa: PR01
+    """Enumerator to Specify the valid values of LogLevel accepted by Logger.setLevel()."""
```

Comment:
```suggestion
    """Enumerator to specify the valid values of LogLevel accepted by Logger.setLevel()."""
```
Why noqa?

Reply:
was getting traceback  in doc_checker.py that {'value', 'names', 'start', 'type', 'module', 'qualname'} not documented, could see other intEnum using #noqa, so used this in this enum as well.